ci/docker-compose.yml

```diff
@@ -30,28 +30,28 @@ services:
       retries: 5
 
   source-test-env:
-    image: public.ecr.aws/x5u3w5h6/rw-build-env:v20230119-1
+    image: public.ecr.aws/x5u3w5h6/rw-build-env:v20230126
     depends_on:
       - mysql
```

Comment:
Hmm, seems that we have dependency on another mysql docker image... was it necessary/is it still necessary?

Reply:
Oh I see, we are using the client to talk to this service? Makes sense.

Reply:
The problem is that we did not update apt source before installing `mysql-client`